requirements/requirements_git.txt

```diff
@@ -1,3 +1,3 @@
 git+https://github.com/ansible/system-certifi.git@devel#egg=certifi
 git+git://github.com/ansible/ansible-runner@devel#egg=ansible-runner
```

Comment:
We should probably update this to the latest stable/major release at some point.